packages/common/package.json

```diff
@@ -11,7 +11,9 @@
         "lint:fix": "eslint . --ext .ts --fix",
         "watch": "tsc --watch",
         "link": "pnpm link --global",
-        "unlink": "pnpm unlink --global"
+        "unlink": "pnpm unlink --global",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u"
```

Comment:
Can we remove this script and use `pnpm run test -- -u` instead when we need to update the snapshots? (We have the script in all our modules, so it's a general observation). I'd rather constrain the number of scripts we have in our `package.json` files. This also ensures that when we update snapshots it's done intentionally and not accidentally (for example, I have autocomplete on my shell and this script makes it easy to inadvertently update snapshots).
```suggestion

```

Reply:
We would need to update the build scripts also, so Ill leave that for @kranthie-sap

Reply:
Lets leave it until we remove them all for consistency.